src/alchemlyb/parsing/gomc.py

```diff
@@ -0,0 +1,211 @@
+"""Parsers for extracting alchemical data from `GOMC <http://gomc.eng.wayne.edu/>`_ output files.
+
+"""
+import pandas as pd
+
+from .util import anyopen
+
+
+# TODO: perhaps move constants elsewhere?
+# these are the units we need for dealing with GOMC, so not
+# a bad place for it, honestly
+# (kB in kJ/molK)
+k_b = 8.3144621E-3
+
+
+def extract_u_nk(filename, T):
+    """Return reduced potentials `u_nk` from a Hamiltonian differences dat file.
+
+    Parameters
+    ----------
+    filename : str
+        Path to free energy file to extract data from.
+    T : float
+        Temperature in Kelvin at which the simulation was sampled.
+
+    Returns
+    -------
+    u_nk : DataFrame
+        Potential energy for each alchemical state (k) for each frame (n).
+
+    """
+
+    dh_col_match = "dU/dL"
+    h_col_match = "DelE"
+    pv_col_match = 'PV'
+    u_col_match = ['Total_En']
+    beta = 1/(k_b * T)
+
+    state, lambdas, statevec = _extract_state(filename)
+
+    # extract a DataFrame from free energy file data
+    df = _extract_dataframe(filename)
+
+    # drop duplicate columns if we (stupidly) have them
+    df = df.iloc[:, ~df.columns.duplicated()]
```

Comment:
I think this can be removed, see recent PR #87 – but double check!

Reply:
In master this line has been removed in gmx.py, but in this PR, it exist still!

Reply:
Yes, because @dotsdl removed it in PR #87.